Core/CoreTests/Features/Files/View/FileList/FileListViewControllerTests.swift

```diff
@@ -271,6 +275,54 @@ class FileListViewControllerTests: CoreTestCase {
         XCTAssertEqual(controller.tableView.numberOfRows(inSection: 0), 0)
     }
 
+    func test_addFileButtonVisible_whenNotRestricted() {
+        controller = FileListViewController.create(
```

Comment:
I think in such cases it's better to explicitly declare the initial state of the test.

```suggestion
        studentAccessInteractor.setRestricted(false)

        controller = FileListViewController.create(
```